src/arizona_js.erl

```diff
@@ -32,16 +32,22 @@ Javascript support.
 %% API funtions.
 %% --------------------------------------------------------------------
 
-send(Event) ->
-    <<"arizona.send.bind(this)('", Event/binary, "')"/utf8>>.
+-spec send(EventName) -> Sent
+    when EventName :: binary(),
+         Sent :: binary().
+send(EventName) ->
+    <<"arizona.send.bind(this)('", EventName/binary, "')"/utf8>>.
 
-send(Event, Payload) ->
-    <<"arizona.send.bind(this)('", Event/binary, "', ", (safe(Payload))/binary, ")"/utf8>>.
+-spec send(EventName, Payload) -> Sent
+    when EventName :: binary(),
+         Payload :: json:encode_value(),
+         Sent :: binary().
+send(EventName, Payload) ->
+    <<"arizona.send.bind(this)('", EventName/binary, "', ", (safe(Payload))/binary, ")"/utf8>>.
 
 %% --------------------------------------------------------------------
 %% Internal funtions.
 %% --------------------------------------------------------------------
 
 safe(Term) ->
     iolist_to_binary(json:encode(Term)).
```

Comment:
In order to keep consistent with `arizona_html:to_safe/1`:
```suggestion
    <<"arizona.send.bind(this)('", EventName/binary, "', ", (to_safe(Payload))/binary, ")"/utf8>>.

%% --------------------------------------------------------------------
%% Internal funtions.
%% --------------------------------------------------------------------

safe(Term) ->
    iolist_to_binary(json:encode(Term)).
```

Reply:
I had to copy-paste this locally to understand what was being changed. It was only `to_safe`. 😄

Reply:
I think you maybe intended to remove the internal (now unused) function, but didn't :) I'll check compilation results locally.

Reply:
I'm reverting this change I committed via GitHub, since I believe something's off. We can come back to it later, in the course of this review.

Reply:
No problem, it's just an internal function.
This suggestion was based on [that change](https://github.com/arizona-framework/arizona/blob/54288eaea9fa38bf8c3490aed23a256d38ee050a/src/arizona_html.erl#L36).